marc_to_solr/lib/traject_config.rb

```diff
@@ -1330,9 +1330,10 @@
 
 # The call_number_locator_display is used in the 'Where to find it' feature in the record page,
 # when the location is firestone$stacks.
+# I dont think we ended up using this field
 to_field 'call_number_locator_display' do |record, accumulator|
   values = []
-  MarcExtractor.cached('852hi').collect_matching_lines(record) do |field, _spec, _extractor|
+  alma_852(record).each do |field|
     subfields = field.subfields.reject { |s| s.value.empty? }.collect { |s| s if ["h", "i"].include?(s.code) }.compact
```

Comment:
Is the `subfields` object used anywhere?

Reply:
It rejects the empty subfields values. But I also think that these field 'call_number_locator_display' is not used at all. I didn't want to remove it now. We created it for the firestone locator because of the 'Oversize' issue but we moved forward with no use of this field.

Reply:
ah you mean the subfields object. rubocop didn't complain so I let it there. I will remove it.

Reply:
event if we dont use it